newsfragments/38071.bugfix.rst

```diff
@@ -0,0 +1,22 @@
+Rename the ``name`` attribute of the StackdriverTaskHandler to ``gcp_log_name`` to avoid name overriding by the the ``DictConfigurator``.
```

Comment:
@VladaZakharova - maybe someone from your team could verify if that kind of change will be breaking or not.